cmd/globalcv/env.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
I'd rather it to be .env file read from go code, so i can use it on windows
eventually we can create powershell file along with bash file

Reply:
works for me